corehq/apps/registration/templates/registration/base.html

```diff
@@ -2,6 +2,23 @@
 
 {% block head %}{{ block.super }}
     {% include 'analytics/fullstory.html' %}
+    {% if FACEBOOK_PIXEL_ID %}
```

Comment:
Does this need to be defined somewhere?

Reply:
https://github.com/dimagi/commcarehq-ansible/pull/955

Reply:
there appears to be middleware taking care of this, as `GOOGLE_ANALYTICS_API_ID` is not defined anywhere in a template context, yet is accessed the same way from a template.

Reply:
I also tested locally and it works.